pybamm/input/parameters/lithium_ion/positive_electrodes/nmc_OKane2022/cracking_rate_Ai2020.py

```diff
@@ -0,0 +1,34 @@
+from pybamm import Parameter, constants, exp
+
+
+def cracking_rate_Ai2020(T_dim):
+    """
+    Particle cracking rate as a function of temperature [1, 2].
+
+    References
+    ----------
+     .. [1] > Ai, W., Kraft, L., Sturm, J., Jossen, A., & Wu, B. (2020).
+     Electrochemical Thermal-Mechanical Modelling of Stress Inhomogeneity in
+     Lithium-Ion Pouch Cells. Journal of The Electrochemical Society, 167(1), 013512
+      DOI: 10.1149/2.0122001JES.
+     .. [2] > Deshpande, R., Verbrugge, M., Cheng, Y. T., Wang, J., & Liu, P. (2012).
+     Battery cycle life prediction with coupled chemical degradation and fatigue
+     mechanics. Journal of the Electrochemical Society, 159(10), A1730.
+
+    Parameters
+    ----------
+    T: :class:`pybamm.Symbol`
+        temperature, [K]
+
+    Returns
+    -------
+    k_cr: :class:`pybamm.Symbol`
+        cracking rate, [m/(Pa.m0.5)^m_cr]
+        where m_cr is another Paris' law constant
+    """
+    k_cr = 3.9e-20
+    Eac_cr = Parameter(
```

Comment:
should be hard-coded

Reply:
Fixed (currently 0, sorry to disappoint!)